test/trento/application/projectors/sap_system_projector_test.exs

```diff
@@ -213,6 +215,85 @@ defmodule Trento.SapSystemProjectorTest do
     )
   end
 
+  test "should broadcast application_instance_absent_at_changed when ApplicationInstanceMarkedAbsent event is received" do
+    insert(:sap_system, id: sap_system_id = Faker.UUID.v4())
+    event = build(:application_instance_registered_event, sap_system_id: sap_system_id)
+
+    ProjectorTestHelper.project(SapSystemProjector, event, "sap_system_projector")
+
+    %{
```

Comment:
This doesn't make sense. You already get this information from the factory, you don't need to re-query.
You can "copy/paste" what you did in the database projection tests